submissions/franchukv/dom-api/script.js

```diff
@@ -0,0 +1,137 @@
+    },
+    'Canopy Piloting': {
+        img: 'img/canopy_piloting.jpg',
+        text: "<p class='content-text__item'>Canopy Piloting is a high speed discipline involving small and very agile parachutes and highly trained pilots to fly them. The competitions are held over a stretch of water for safety reasons and can be watched from just a few meters away from the ground. The athletes accelerate their parachutes by flying one or more steep turns and then plain out over the surface of the water to enter the course. Three classic disciplines define the champion of canopy piloting, that is Accuracy, Speed and Distance.</p>",
+    },
+};
+
+function createSideNavigation() {
+    if (!mainContent.classList.contains('non-active')) {
+        footer.classList.add('footer--element-end');
+    }
+
+    startNav.classList.add('side-menu');
+
+    startHeader.classList.remove('active');
+    startNav.classList.remove('start-menu');
+    resetBtn.classList.remove('non-active');
+    burgenBtn.classList.remove('non-active');
+    mainContent.classList.remove('non-active');
+}
+
+function goToStartPage() {
+    startHeader.classList.add('active');
+    startNav.classList.add('_start--marker');
+    startNav.classList.add('start-menu');
+    resetBtn.classList.add('non-active');
+    burgenBtn.classList.add('non-active');
+    mainContent.classList.add('non-active');
+
+    startNav.classList.remove('side-menu');
+    burgenIcon.classList.remove('burger-icon--active');
+    document.body.classList.remove('_scroll-lock');
+    footer.classList.remove('footer--element-end');
+}
+
+function burgerHandler() {
+    startNav.classList.toggle('side-menu--active');
+    burgenBtn.classList.toggle('burger--white');
+    burgenIcon.classList.toggle('burger-icon--active');
+    document.body.classList.toggle('_scroll-lock');
+}
+
+function navigationHandler(event) {
+    if (startHeader.classList.contains('active') && startNav.classList.contains('start-menu')) {
+        createSideNavigation();
+        createContent(event.target.textContent);
+    }
+
+    if (startNav.classList.contains('_start--marker')) {
+        startNav.classList.remove('side-menu--active');
+        startNav.classList.remove('_start--marker');
+        burgenBtn.classList.remove('burger--white');
+        burgenIcon.classList.remove('burger-icon--active');
+    }
+
+    if (startNav.classList.contains('side-menu--active')) {
+        startNav.classList.remove('side-menu--active');
+        startNav.classList.remove('_start-marker');
+        burgenBtn.classList.remove('burger--white');
+        document.body.classList.remove('_scroll-lock');
+        burgenIcon.classList.remove('burger-icon--active');
+    }
+
+    createContent(event.target.textContent);
+}
+
+function createNavigation(item) {
+    for (item in data) {
+        let navItem = document.createElement('li');
+        navItem.classList.add('cards__item');
+
+        let h3 = document.createElement('h3');
```

Comment:
In your navigation user clicks on the heading. The heading is not an interactive element, use the corresponding HTML tag for that.